packages/database/src/core/util/libs/parser.ts

```diff
@@ -134,6 +136,11 @@ export const parseURL = function(
     scheme = 'https',
     port = 443;
 
+  // Don't use SSL if we are parsing an emulator URL.
+  if (process.env[FIREBASE_DATABASE_EMULATOR_HOST_VAR]) {
+    scheme = 'http';
```

Comment:
This seems unnecessary, aren't we directly extracting the address in the other file modified in this PR?

Reply:
To keep consistent with `FIRESTORE_EMULATOR_HOST`, the `FIREBASE_DATABASE_EMULATOR_HOST` must not specify a protocol. Before some changes to the node.js admin sdk which append the protocol identifier before passing a url to this module, this code was needed. It is no longer needed.